pkg/sql/schema_changer_test.go

```diff
@@ -6238,6 +6238,14 @@ ALTER TABLE t.public.test DROP COLUMN v;
 	require.Equal(t, [][]string{
 		{"1", "2"},
 	}, rows)
+
+	// Validate the job cancellation metrics.
+	rows = runner.QueryStr(t, "SELECT * FROM crdb_internal.feature_usage WHERE feature_name LIKE 'job.%.canceled'")
```

Comment:
Perhaps I misunderstood the issue but is there a specific reason why you're querying the telemetry table instead of `SELECT COUNT(*) FROM system.jobs WHERE status = 'canceled'`?

Reply:
The intent of these changes is to only provide coverage for the telemetry. I didn't add any coverage for validating the cancelled counters for jobs.